### PR TITLE
fix(repayment): get filtered repayment for servicename

### PR DESCRIPTION
**Pull request by Heiiji**
| Question         | Answer
| ---------------- | ---
| Branch?          | `develop`
| Bug fix?         | yes
| New feature?     | no
| Breaking change? | no
| Tickets          | MANAGER-6854
| License          | BSD 3-Clause

## Description

Use new query parameter of repayment 2api endpoint to get a filtered result for a specific number (serviceName).

### CODE REVIEW

File: packages/manager/apps/telecom/src/app/telecom/telephony/alias/special/repayments/repayments.controller.js [hunk 1]
```diff
@@ -31,13 +39,16 @@ export default class TelecomTelephonyAliasSpecialRepaymentsCtrl {
 
     this.loading = true;
 
-    return this.OvhApiTelephony.Service()
```
Comment (antleblanc) [inline packages/manager/apps/telecom/src/app/telecom/telephony/alias/special/repayments/repayments.controller.js:34]: `OvhApiTelephony` still needed?

File: packages/manager/apps/telecom/src/app/telecom/telephony/alias/special/repayments/repayments.controller.js [hunk 2]
```diff
@@ -31,13 +39,16 @@ export default class TelecomTelephonyAliasSpecialRepaymentsCtrl {
...
-      .RepaymentConsumption()
-      .Aapi()
-      .repayment({
+    return this.OvhHttp.get('/telephony/{billingAccount}/repayment', {
```
Comment (antleblanc) [inline packages/manager/apps/telecom/src/app/telecom/telephony/alias/special/repayments/repayments.controller.js:42]: Both `OvhHttp` and `OvhApi*` services are deprecated.

We tend to use `$http`. Is it possible to use it here?

**Review by antleblanc — CHANGES_REQUESTED** [review]

File: packages/manager/apps/telecom/src/app/telecom/telephony/alias/special/repayments/repayments.controller.js [hunk 3]
```diff
@@ -6,11 +6,12 @@ import { LIMIT } from './repayments.constants';
 
 export default class TelecomTelephonyAliasSpecialRepaymentsCtrl {
   /* @ngInject */
-  constructor($state, $stateParams, coreConfig, TucToast, OvhApiTelephony) {
+  constructor($state, $stateParams, $http, OvhHttp, coreConfig, TucToast) {
+    this.OvhHttp = OvhHttp;
```
Comment (marie-j) [inline packages/manager/apps/telecom/src/app/telecom/telephony/alias/special/repayments/repayments.controller.js:10]:
```suggestion
```

**Review by marie-j — APPROVED** [review]

**Review by frenautvh — APPROVED** [review]

**Review by jasi232 — APPROVED** [review]

**Review by JeremyDec — APPROVED** [review]

File: packages/manager/apps/telecom/src/app/telecom/telephony/alias/special/repayments/repayments.controller.js (hunk 3 above)
Comment (jasi232) [reply packages/manager/apps/telecom/src/app/telecom/telephony/alias/special/repayments/repayments.controller.js]: Dont do it to ur self

**Comment by jasi232** [comment]
Don't do it to ur self stupid

**Review by antleblanc — APPROVED** [review]

**Pull request merged**